index.html

```diff
@@ -966,7 +966,8 @@ <h2><dfn>Coalesced events</dfn></h2>
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
             zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
             <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
-            that were coalesced into this event. For all other trusted event types, it is an empty list.
+            that were coalesced into this event, including a clone of the "parent" event itself.
```

Comment:
do we need to maybe even specify that the clone is the *last* event in the list?

Reply:
The order is defined in the text below.